common/src/main/scala/explore/components/undo/Undoer.scala

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components.undo
+
+import react.common.ReactProps
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import cats.effect.IO
+import cats.implicits._
+import monocle.macros.Lenses
+import crystal.react.io.implicits._
+import monocle.Lens
+import monocle.Getter
+
+final case class Undoer[M](
```

Comment:
I guess this will wrap an undo context?

Reply:
Exactly. We can have different undo regions if we want.